src/TaskManager/TaskManager.cs

```diff
@@ -96,18 +96,22 @@ private async Task TaskCallbackEventReceivedCallback(MessageReceivedEventArgs ar
         {
             Guard.Against.Null(args, nameof(args));
             using var loggingScope = _logger.BeginScope($"Message Type={args.Message.MessageDescription}, ID={args.Message.MessageId}. Correlation ID={args.Message.CorrelationId}");
-            var message = args.Message.ConvertToJsonMessage<TaskCallbackEvent>();
+
+            JsonMessage<TaskCallbackEvent>? message = null;
             try
             {
+                message = args.Message.ConvertToJsonMessage<TaskCallbackEvent>();
                 await HandleTaskCallback(message).ConfigureAwait(false);
             }
             catch (OperationCanceledException ex)
             {
                 _logger.ServiceCancelledWithException(ServiceName, ex);
+                await _messageBrokerSubscriberService!.RequeueWithDelay(message);
```

Comment:
can ConvertToJsonMessage fail here?

Reply:
yeah it could, i've pulled this from the args directly now